README.md

```
@@ -1,235 +1,378 @@
+- 🎥 [Video walkthrough by BugBytes](https://www.youtube.com/watch?v=Pukr-fT4MFY)
+- 🤖 [Ask DeepWiki](https://deepwiki.com/FarhanAliRaza/django-bolt) · for AI assistants: [llms.txt](https://bolt.farhana.li/llms.txt)
+- ❓ [FAQ](https://bolt.farhana.li/faq/) · [Comparison vs Ninja / DRF / FastAPI / Litestar](https://bolt.farhana.li/comparison/) · [How it works](https://bolt.farhana.li/architecture/)
 
-- **[Litestar](https://github.com/litestar-org/litestar)** - Our OpenAPI plugin system is adapted from Litestar's excellent architecture. Many architectural decisions around middleware, guards, and route handling were inspired by Litestar's design philosophy.
+## 💖 Sponsors
 
-- **[Robyn](https://github.com/sparckles/Robyn)** - Robyn's Rust-Python integration patterns and performance-first approach influenced our decision to use PyO3 and showed us the potential of Rust-powered Python web frameworks.
+Support Django-Bolt's development by [becoming a sponsor](https://opencollective.com/django-bolt). Your logo will appear here with a link to your website.
 
-### Additional Credits
+<a href="https://opencollective.com/django-bolt/tiers/sponsor/0/website" target="_blank"><img src="https://opencollective.com/django-bolt/tiers/sponsor/0/avatar.svg" /></a>
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)